contracts/libraries/TridentMath.sol

```diff
@@ -61,4 +61,8 @@ library TridentMath {
             }
         }
     }
+
```

Comment:
We should probably avoid this. It will mean bytecode changing in CPP again, we already have 2 different bytecodes to deal with now.

Reply:
Yes, will move this to other lib along with other fixes.